### PR TITLE
Update bundle_zh_CN.properties

### PR DESCRIPTION
**Pull request by xem8k5**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle_zh_CN.properties [hunk 1]
```diff
@@ -1921,96 +1921,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
-block.reinforced-payload-router.description = Distributes payloads into adjacent blocks. Functions as a sorter when a filter is set.
-block.payload-mass-driver.description = Long-range payload transport structure. Shoots received payloads to linked payload mass drivers.
-block.unit-repair-tower.description = Repairs all units in its vicinity. Requires ozone.
-block.radar.description = Gradually uncovers terrain and enemy units in a large radius. Requires power.
-block.shockwave-tower.description = Damages and destroys enemy projectiles in a radius. Requires cyanogen.
+block.core-bastion.description = 基地的核心。 拥有装甲。 一旦被摧毁，此区块就会丢失。 
+block.core-citadel.description = 基地的核心。 装甲优良。 容量比城堡核心更大。 
+block.core-acropolis.description = 基地的核心。 装甲极佳。 容量比堡垒核心更大。 
+block.breach.description = 向敌人发射铍或钨穿透子弹。
+block.diffuse.description = 向敌人发射锥形爆裂子弹。 并把敌人击退。
```
Comment (Cry0flu1d) [inline core/assets/bundles/bundle_zh_CN.properties:1928]: should be "向前方锥形区域发射子弹"

File: core/assets/bundles/bundle_zh_CN.properties [hunk 2]
```diff
@@ -1921,96 +1921,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.sublimate.description = 向敌人持续喷射火焰。 穿甲。
```
Comment (Cry0flu1d) [inline core/assets/bundles/bundle_zh_CN.properties:1929]: 穿透敌方装甲

File: core/assets/bundles/bundle_zh_CN.properties [hunk 3]
```diff
@@ -1921,96 +1921,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.reinforced-liquid-junction.description = 两条交叉强化导管的桥梁。
+block.surge-conveyor.description = 打包物品进行运输。 可以用电力加速。 需要电力。
+block.surge-router.description = 从合金传送带上的物品平均分配到其它3个方向。 可以用电力加速。 需要电力。
+block.unit-cargo-loader.description = 建造货运无人机。 无人机通过需求自动将物品分配到单位物流卸载点。
+block.unit-cargo-unload-point.description = 作为货运无人机的卸货点。接受需求的物品。
+block.beam-node.description = 向其他互相垂直的节点传输电力。 存储少量电力。
+block.beam-tower.description = 向其他互相垂直的节点传输电力。 存储少量电力。 连接距离远。
+block.turbine-condenser.description = 当放置在喷口上时产生电力。 同时产生少量水。
+block.chemical-combustion-chamber.description = 利用芳油和臭氧发电。
+block.pyrolysis-generator.description = 通过芳油和矿渣产生大量电力。 同时产生水。
```
Comment (Cry0flu1d) [inline core/assets/bundles/bundle_zh_CN.properties:1989]: 使用矿渣热解芳油产生大量电力

File: core/assets/bundles/bundle_zh_CN.properties [hunk 4]
```diff
@@ -2050,24 +2050,24 @@ unit.oxynoe.description = 向附近的敌人发射火焰束，并修复己方建
...
+unit.tecta.description = 向敌人发射等离子追踪导弹。 用它前方的护盾保护自己。 可以跨越大多数地形。 
+unit.collaris.description = 向敌人远距离发射分裂火炮。 可以跨越大多数地形。 
+unit.elude.description = 向敌人发射螺旋状追踪子弹。可以在液体上悬浮。
+unit.avert.description = 向敌人发射螺旋状扭曲子弹。
+unit.obviate.description = 向敌人发射螺旋状闪电球。
+unit.quell.description = 向敌人远距离发射追踪导弹。 能够压制敌方修复建筑。
+unit.disrupt.description = 向敌人远距离发射压制性追踪导弹。 能够压制敌方修复建筑。
+unit.evoke.description = 保护城堡核心，可建造建筑。 使用激光束修复建筑。
+unit.incite.description = 保护堡垒核心，可建造建筑。 使用激光束修复建筑。
+unit.emanate.description = 保护卫城核心，可建造建筑。 使用成对的激光束修复建筑。
```
Comment (Cry0flu1d) [inline core/assets/bundles/bundle_zh_CN.properties:2070]: “一对”, not “成对的”

File: core/assets/bundles/bundle_zh_CN.properties [hunk 5]
```diff
@@ -1921,96 +1921,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.cliff-crusher.description = 粉碎墙壁，以缓慢的速度无限产出沙子。 这需要电力。 效率因墙的类型而异。
+block.impact-drill.description = 放置在矿物上时，以缓慢的速度无限产出物品。 需要电力和水。
+block.eruption-drill.description = 改进过的冲击钻头。 能够开采钍。 需要氢。
+block.reinforced-conduit.description = 向前传输流体。 不接受侧面的非导管输入。
+block.reinforced-liquid-router.description = 将流体平均分配到所有侧面方向。
+block.reinforced-junction.description = 两条交叉物品管道的桥梁。
+block.reinforced-liquid-tank.description = 储存大量的流体。
+block.reinforced-liquid-container.description = 储存数量可观的流体。
+block.reinforced-bridge-conduit.description = 跨越任意地形或建筑物传输流体。
+block.reinforced-pump.description = 泵送液体。 需要氢气
```
Comment (Cry0flu1d) [inline core/assets/bundles/bundle_zh_CN.properties:1962]: missing period

File: core/assets/bundles/bundle_zh_CN.properties [hunk 6]
```diff
@@ -747,20 +747,20 @@ sector.lake.name = 岩浆湖
...
-sector.basin.name = 芳油盆地
-sector.marsh.name = Marsh
+sector.basin.name = 风蚀盆地
+sector.marsh.name = 芳油湿地
 sector.peaks.name = Peaks
-sector.ravine.name = Ravine
-sector.caldera-erekir.name = Caldera
+sector.ravine.name = 贫瘠峡谷
+sector.caldera-erekir.name = 破碎火山
 sector.onset.description = 教学区块。区块目标未设立，等待进一步指示。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:755]: 句号后的空格。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 7]
```diff
@@ -1759,22 +1759,22 @@ item.spore-pod.description = 用于转化成石油、 爆炸混合物或作为
 item.spore-pod.details = 孢子。 可能是人工合成的生命形式，释放对其他生物有毒的气体，极具侵略性。 在某些条件下高度易燃。 
 item.blast-compound.description = 用作炸弹或爆炸性弹药。 
 item.pyratite.description = 用于火焰发射类武器和使用燃料的发电设备。 
-item.beryllium.description = Used in many types of construction and ammunition on Erekir.
-item.tungsten.description = Used in drills, armor and ammunition. Required in the construction of more advanced structures.
-item.oxide.description = Used as a heat conductor and insulator for power.
-item.carbide.description = Used in advanced structures, heavier units, and ammunition.
+item.beryllium.description = 用于埃里克尔的多种建筑和弹药。
+item.tungsten.description = 用于钻头、装甲和弹药。 或用于建造更先进的结构。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:1763]: 顿号后的空格。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 8]
```diff
@@ -1759,22 +1759,22 @@ item.spore-pod.description = 用于转化成石油、 爆炸混合物或作为
...
+item.oxide.description = 用作热导体和绝缘体。
+item.carbide.description = 用于先进的结构、重型单位和弹药。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:1765]: 空格。结构->建筑？

File: core/assets/bundles/bundle_zh_CN.properties [hunk 9]
```diff
@@ -1759,22 +1759,22 @@ item.spore-pod.description = 用于转化成石油、 爆炸混合物或作为
...
-liquid.nitrogen.description = Used in resource extraction, gas creation and unit production. Inert.
-liquid.neoplasm.description = A dangerous biological byproduct of the Neoplasia reactor. Quickly spreads to any adjacent water-containing block it touches, damaging them in the process. Viscous.
-liquid.neoplasm.details = Neoplasm. An uncontrollable mass of rapidly-dividing synthetic cells with a sludge-like consistency. Heat-resistant. Extremely dangerous to any structures involving water.\n\nToo complex and unstable for standard analysis. Potential applications unknown. Incineration in slag pools is recommended.
+liquid.arkycite.description = 用于发电和合成材料的化学反应。
+liquid.ozone.description = 用作材料生产的氧化剂和燃料。 有一定爆炸性。
+liquid.hydrogen.description = 用于资源开采、 单位生产和结构修复。 易燃。
+liquid.cyanogen.description = 用于构建高级建筑和单位，也用作弹药。 高度易燃。
+liquid.nitrogen.description = 用于资源开采、 气体生产和单位生产。
+liquid.neoplasm.description = 瘤变反应堆产出的危险生物性副产物。 会很快扩散至它能接触到的含水方块并破坏它们。 有粘性。
+liquid.neoplasm.details = 瘤液。 由一种无法控制并快速分裂的合成细胞团组成，具有污泥状的粘稠度。 耐热。 对任何含有水的结构极为危险。 \n\n从标准分析来看过于复杂和不稳定，潜在的应用未知。 建议在矿渣池中焚毁
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:1777]: \n前的空格删去。句末的句号加上。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 10]
```diff
@@ -1925,96 +1925,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.atmospheric-concentrator.description = 从大气中浓缩氮气。 需要热量。
+block.surge-crucible.description = 将矿渣和硅合成巨浪合金。 需要热量。
+block.phase-synthesizer.description = 将钍、沙子和臭氧合成相织布。 需要热量。
+block.carbide-crucible.description = 将石墨和钨熔化为碳化物。 需要热量。
+block.cyanogen-synthesizer.description = 将芳油和石墨合成氰。 需要热量。
+block.slag-incinerator.description = 焚烧稳定的物品和液体。 需要矿渣。
+block.vent-condenser.description = 将排出的水蒸气冷凝成水。 消耗电力。 
+block.plasma-bore.description = 面向矿壁放置时，以缓慢的速度无限产出物品。 需要少量电力。
+block.large-plasma-bore.description = 更大的等离子钻机。 能够开采钨和钍。 需要氢气和电力。 
+block.cliff-crusher.description = 粉碎墙壁，以缓慢的速度无限产出沙子。 这需要电力。 效率因墙的类型而异。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:1957]: 《这》

File: core/assets/bundles/bundle_zh_CN.properties [hunk 11]
```diff
@@ -1925,96 +1925,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.tungsten-wall-large.description = 保护己方建筑，挡下敌方炮弹。
+block.carbide-wall.description = 保护己方建筑，挡下敌方炮弹。
+block.carbide-wall-large.description = 保护己方建筑，挡下敌方炮弹。
+block.reinforced-surge-wall.description = 保护己方建筑，挡下敌方炮弹。 受攻击时间断释放电弧。
+block.reinforced-surge-wall-large.description = 保护己方建筑，挡下敌方炮弹。 受攻击时间断释放电弧。
+block.shielded-wall.description = 保护己方建筑，挡下敌方炮弹。 创建一个力场，并在通电时吸收大部分炮弹。 需要电力。
+block.blast-door.description = 当己方地面单位在范围内时自动打开的墙。 无法手动控制。
+block.duct.description = 向前运输物品。 只能存储一个物品。
+block.armored-duct.description = 向前运输物品。 不接受侧面的非管道输入。
+block.duct-router.description = 将物品平均分配到其它3个方向。 只接受背面的物品。 可被配置为分类器
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:1979]: 句号。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 12]
```diff
@@ -1925,96 +1925,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.mech-fabricator.description = 生产机甲单位。 产出的单位可以直接投入战场，也可以送入重构工厂进行升级。 
+block.tank-assembler.description = 利用输入的方块和单位组装大型坦克。 可以通过添加基本装配厂模块来增加输出等级。
+block.ship-assembler.description = 利用输入的方块和单位组装大型飞船。 可以通过添加基本装配厂模块来增加输出等级。
+block.mech-assembler.description = 利用输入的方块和单位组装大型机甲。 可以通过添加基本装配厂模块来增加输出等级。
+block.tank-refabricator.description = 将输入的坦克单位升级到第二级。 
+block.ship-refabricator.description = 将输入的飞船单位升级到第二级。 
+block.mech-refabricator.description = 将输入的机甲单位升级到第二级。 
+block.prime-refabricator.description = 将输入的单位升级到第三级。 
+block.basic-assembler-module.description = 当放置在构造边界旁边时，增加输出等级。 需要电力。 可以用作载荷输入。
+block.small-deconstructor.description = 解构方块和单位，返还100%资源
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:2011]: 。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 13]
```diff
@@ -1925,96 +1925,96 @@ block.large-logic-display.description = 显示处理器中绘制的各种图形
...
+block.reinforced-payload-conveyor.description = 向前运输载荷。
+block.reinforced-payload-router.description = 将载荷向3个方向轮流输出。设置载荷时可用作分类器。 
+block.payload-mass-driver.description = 远距离荷载运输建筑。 向相连的其它载荷质量驱动器发射载荷
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:2014]: 。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 14]
```diff
@@ -2054,24 +2054,24 @@ unit.oxynoe.description = 向附近的敌人发射火焰束，并修复己方建
...
-unit.quell.description = Fires long-range homing missiles at enemy targets. Suppresses enemy structure repair blocks.
-unit.disrupt.description = Fires long-range homing suppression missiles at enemy targets. Suppresses enemy structure repair blocks.
-unit.evoke.description = Builds structures to defend the Bastion core. Repairs structures with a beam.
-unit.incite.description = Builds structures to defend the Citadel core. Repairs structures with a beam.
-unit.emanate.description = Builds structures to defend the Acropolis core. Repairs structures with beams.
+unit.stell.description = 向敌人发射普通子弹。
+unit.locus.description = 向敌人交替发射子弹。
+unit.precept.description = 向敌人发射穿透性集束子弹。
+unit.vanquish.description = 向敌人发射大型穿透性分裂子弹。
+unit.conquer.description = 向敌人发射大量穿透子弹。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:2061]: 穿透“性”子弹。统一叫法。

File: core/assets/bundles/bundle_zh_CN.properties [hunk 15]
```diff
@@ -2054,24 +2054,24 @@ unit.oxynoe.description = 向附近的敌人发射火焰束，并修复己方建
...
+unit.merui.description = 向地面敌人发射远程火炮。 可以跨越大多数地形。 
+unit.cleroi.description = 向敌人发射双发炮弹。 搭载一台单点防御炮塔，能够防御来袭的子弹。 可以跨越大多数地形。 
+unit.anthicus.description = 向敌人远距离发射追踪导弹。 可以跨越大多数地形。 
+unit.tecta.description = 向敌人发射等离子追踪导弹。 用它前方的护盾保护自己。 可以跨越大多数地形。 
+unit.collaris.description = 向敌人远距离发射分裂火炮。 可以跨越大多数地形。 
+unit.elude.description = 向敌人发射螺旋状追踪子弹。可以在液体上悬浮。
```
Comment (SoMall-dumpling) [inline core/assets/bundles/bundle_zh_CN.properties:2067]: 句号后的空格。

**Review by SoMall-dumpling — COMMENTED** [review]
以上是初步浏览的情况。翻译的改进以后再说吧。

**Comment by xem8k5** [comment]
@Anuken done

**Comment by SoMall-dumpling** [comment]
@Anuken merge pls

**Pull request merged**